src/rpc/server.h

```diff
@@ -128,13 +128,29 @@ void RPCRunLater(const std::string& name, std::function<void(void)> func, int64_
 
 typedef UniValue(*rpcfn_type)(const JSONRPCRequest& jsonRequest);
 
+typedef enum{
```

Comment:
Use `enum class`.

Reply:
Also, if you use `enum class`, get rid of the `eRPCCategory_` prefix (since then it will be namespaced already).

Reply:
Please don't add bitcoin-specific things (such as these categories) in `rpc/server.h`, the base RPC code is supposed to be independent of any specific use.

Reply:
Using `enum class` in https://github.com/bitcoin/bitcoin/pull/13945/commits/cf190f57a078df1d37bd1822e3fa69ba0d9c90e6